src/font.c

```diff
@@ -1074,9 +1078,12 @@ GdipPrivateAddMemoryFont(GpFontCollection *fontCollection, GDIPCONST void *memor
 	if (!memory)
 		return InvalidParameter;
 
+#ifndef WIN32
 	strcpy((char *) fontfile, "/tmp/ffXXXXXX");
+#endif
 
 	f = mkstemp((char*)fontfile);
```

Comment:
I'd like to avoid calling the Win32 reimplementation "mkstemp" when it doesn't follow the semantics of mkstemp. 

Let's call it e.g. `CreateTempFile()` or something like that.